libs.gradle

```diff
@@ -1,7 +1,7 @@
 def libs = ext.libs = [:]
 
 libs.axelor_studio = 'com.axelor.addons:axelor-studio:2.0.1'
-libs.axelor_message = 'com.axelor.addons:axelor-message:2.0.0'
+libs.axelor_message = 'com.axelor.addons:axelor-message:3.0.0'
 libs.axelor_utils = 'com.axelor.addons:axelor-utils:2.0.0'
```

Comment:
shouldn't you update the 3 libs to 3.0.0? I think the major version are supposed to stay aligned